### ISSUE TITLE
Operation with zippers

### ISSUE THREAD

**Issue by phreed** [nathanmarz/specter#62]
It seems that specter should would well with zippers.
That is zippers are good at relative movement and updates and specter does well with (absolute) paths.
My questions are:
Does specter interoperate with zippers?
Do you have any examples?
Are there any pitfalls?

My present case involves walking the zipper and running a multi-function on each node.
The default function is to leave the node unchanged but if it meets certain criteria 
then I want to update the zipper. The zipper is implemented as a PersistentVector pair.
The first element is the child tree and it seems like any specter path into that tree should be fine.
What do you think?

**Comment by nathanmarz** [comment]
It should be very easy to make your own StructurePath implementations that integrate zippers into Specter. Basically you would want to have a path to navigate to a zipper, and then a path for each of the zipper operations. So then you could write something like:

(transform [z/VECTOR z/DOWN z/RIGHT z/DOWN z/RIGHT z/NODE z/ALL] inc [1 [2 [3 4]] 5])
;; => [1 [2 [4 5]] 5]

**Comment by nathanmarz** [comment]
It's so easy to integrate that I did it in a quick gist: https://gist.github.com/nathanmarz/9981c0b04e712d460a21

My example above becomes:

```clojure
(transform [z/VECTOR-ZIP z/DOWN z/RIGHT z/DOWN z/RIGHT z/NODE z/ALL] inc [1 [2 [3 4]] 5])
;; => [1 [2 [4 5]] 5]
```

**Comment by nathanmarz** [comment]
Zippers integration was released with 0.10.0
